### DISCUSSION TITLE
Add a tutorial to deploy to Google App Engine using MySQL

### DISCUSSION THREAD

**Discussion by LucasAndrad** [strapi/strapi#6116] [Legacy Topics]
Not sure if many people require that, but it would be nice to also have a tutorial (or even just a new section) teaching how to deploy to google engine using MySQL.

**Comment by lauriejim** [comment]
Hello, here is into the docs the documentation for GCP
https://strapi.io/documentation/3.0.0-beta.x/deployment/google-app-engine.html
But it is using PG, I suppose it's pretty much the same for MySQL (just the database part to adapt I guess)

**Reply by k-hud** [reply]
@lauriejim I just opened a discussion [about this here](https://github.com/strapi/strapi/discussions/7338).

The GCP App Engine walkthrough uses --quickstart which defaults to SQLlite, but then proceeds to talk through the PostgreSQL setup. There are a few gaps in the fact that by setting everything up for PostgreSQL, it doesn't address maintaining SQL for the development environment, or setting up development for PostgreSQL so you can locally build and start. 

Any direction you can provide would be amazing, I'd love to help make the documentation better, but have spent a few days debugging trying to find the right path. 

Thanks again and kudos on the amazing product, really enjoying it so far.

**Reply by lauriejim** [reply]
We will answer on the other discussion thread.

**Comment by i99dev** [comment]
I try to work with MTSQL from SQL SERVER GCLOUD, the problem is I can't contact with a database through the socket and try through TCP no one work with us.

and still looking for solutions, if anybody setup strapi with my SQL on app engine please share with us the steps
